web/src/frontend/integration/test/hovedtest.js

```diff
@@ -0,0 +1,36 @@
+var hovedside, timeout;
+
+module.exports = {
+	before: (browser) => {
+        timeout = browser.globals.test_settings.timeout;
+        const browserInit = browser.init();
+        hovedside = browserInit.page.hovedsidepage();
+        hovedside.navigate();
+    },
+    after: (browser) => {
+        browser.end();
+    },
+    "hovedside skal ha minst ett skjemaelement": function () {
+        hovedside.expect.element('@input').to.be.present.after(timeout);
+    },
+    "hovedside skal ha app tittel": function () {
+		hovedside.expect.element('@appTitle').to.be.present.after(timeout);
+    },
+	"hovedside skal ikke ha fortsett knapp hvis man ikke har valgt bosted": function () {
+		hovedside.expect.element('@fortsett').to.not.be.present.after(timeout);
+	},
+	"hovedside skal ha en fortsett knapp når man har valgt kommune": function () {
+		hovedside.click('@horten');
+		hovedside.expect.element('@fortsett').to.be.present.after(timeout);
+		hovedside.click('@fortsett');
```

Comment:
kanskje dette steget skulle vært en egen test som heter noe som at man skal få startet en søknad? Man må kjenne designet godt for å skjønne at fortsett betyr opprett søknad ;)

Reply:
Enig. Testen heter nå ```"man skal få opprettet en søknad når man har valgt kommune og klikket på Fortsett knappen"```.